lib/alephant/broker/response/base.rb

```diff
@@ -14,6 +14,7 @@ class Base
 
         STATUS_CODE_MAPPING = {
           200 => "ok",
+          304 => nil,
```

Comment:
I thought that at first, but the mapping is used as the response, then seemingly overwritten for the 200, so it only really maps error content.

So I have now set it to an empty string, as a 304 usually has no content returned. See comment about this, as the content type now changes...